frontend/src/components/Scenario/CompartmentList.tsx

```diff
@@ -0,0 +1,243 @@
+  const compartmentValues = useGetSimulationStartValues();
+
+  /** This function either returns the value at simulation start of a compartment or 'no data'. */
+  const getCompartmentValue = (compartment: string): string => {
+    if (compartmentValues && compartment in compartmentValues) {
+      return formatNumber(compartmentValues[compartment]);
+    }
+    return t('no-data');
+  };
+
+  // This effect sets the selected compartment to the first one if no compartment is initially selected.
+  useEffect(() => {
+    if (!selectedCompartment && compartments.length > 0) {
+      dispatch(selectCompartment(compartments[0]));
+    }
+  }, [dispatch, compartments, selectedCompartment]);
+
+  return (
+    <Box
+      id='scenario-view-compartment-list-root'
+      sx={{
+        borderRight: `2px dashed ${theme.palette.divider}`,
+        flexGrow: 0,
+        flexShrink: 0,
+        flexBasis: '274px',
+        minHeight: '20vh',
+        maxWidth: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        marginTop: theme.spacing(3),
+        borderTop: '2px solid transparent', // invisible border for alignment with the scenario card
+        paddingBottom: 0,
+        paddingTop: theme.spacing(2),
+        paddingLeft: 0,
+        paddingRight: 0,
+      }}
+    >
+      <SimulationStartTitle />
+      <ScrollSyncPane group='compartments'>
+        <List
+          id='scenario-view-compartment-list'
+          dense={true}
+          disablePadding={true}
+          sx={{
+            maxHeight: compartmentsExpanded ? '248px' : 'auto',
+            overflowY: 'auto',
+          }}
+        >
+          {compartments.map((compartment, i) => (
+            <CompartmentRow
+              key={compartment}
+              compartment={compartment}
+              value={getCompartmentValue(compartment)}
+              index={i}
+            />
+          ))}
+        </List>
+      </ScrollSyncPane>
+      <Button
+        variant='outlined'
+        color='primary'
+        sx={{
+          margin: theme.spacing(3),
+          marginTop: theme.spacing(4),
+          marginBottom: 0,
+          padding: theme.spacing(1),
+        }}
+        aria-label={t('scenario.more')}
+        onClick={() => {
+          dispatch(toggleCompartmentExpansion());
+          if (
+            compartments.findIndex((o) => {
+              return o === selectedCompartment;
+            }) > 4
+          ) {
+            if (compartments.length > 0) {
+              dispatch(selectCompartment(compartments[0]));
+            }
+          }
+        }}
+      >
+        {compartmentsExpanded ? t('less') : t('more')}
+      </Button>
+    </Box>
+  );
+}
+
+/** This component renders the simulation start date together with a descriptive label. */
+function SimulationStartTitle(): JSX.Element {
+  const theme = useTheme();
+  const {t, i18n} = useTranslation();
+
+  const startDay = useAppSelector((state) => state.dataSelection.minDate);
```

Comment:
Not entirely sure if this works properly.
Day needs to be adjusted.
```suggestion
  const startDay = useAppSelector((state) => {
    let date = new Date(state.dataSelection.minDate);
    // start day is the day before the first simulation result
    date = new Date(date.getDate() - 1);
    return `${date.toISOString()[0:9]}`
  });
```